src/tablecellproperties/commands/tablecellpropertycommand.js

```diff
@@ -98,4 +95,28 @@ export default class TableCellPropertyCommand extends Command {
 	_getValueToSet( value ) {
 		return value;
 	}
+
+	/**
+	 * Returns a single value for selected table cells. If all cells have the same value set it returns that value or undefined otherwise.
```

Comment:
```suggestion
	 * Returns a single value for all selected table cells. If the value is the same for all cells, it will be returned (`undefined` otherwise).
```